apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjFissDiagnosisCode.java

```diff
@@ -24,23 +24,23 @@
 @NoArgsConstructor
 @EqualsAndHashCode(onlyExplicitlyIncluded = true)
 @FieldNameConstants
-@IdClass(PreAdjFissDiagnosisCode.PK.class)
 @Table(name = "`FissDiagnosisCodes`", schema = "`pre_adj`")
+@IdClass(PreAdjFissDiagnosisCode.PK.class)
 public class PreAdjFissDiagnosisCode {
   @Id
-  @Column(name = "`dcn`", length = 23, nullable = false)
   @EqualsAndHashCode.Include
+  @Column(name = "`dcn`", nullable = false, length = 23)
   private String dcn;
 
   @Id
-  @Column(name = "`priority`", nullable = false)
   @EqualsAndHashCode.Include
+  @Column(name = "`priority`", nullable = false)
   private short priority;
 
-  @Column(name = "`diagCd2`", length = 7, nullable = false)
+  @Column(name = "`diagCd2`", nullable = false, length = 7)
   private String diagCd2;
 
-  @Column(name = "`diagPoaInd`", length = 1, nullable = false)
+  @Column(name = "`diagPoaInd`", nullable = false, length = 1)
```

Comment:
I thought the RDA API team changed this to be nullable?  https://confluenceent.cms.gov/pages/viewpage.action?pageId=233814218

Reply:
Great catch!